src/components/style.css

```diff
@@ -0,0 +1,50 @@
+@font-face {
+  font-family: 'digital';
+  src: url(../digital-7.ttf);
+}
+
+.app,
+.button-panel {
+  display: flex;
+  flex-flow: column;
+}
+
+.app {
+  width: 700px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.display {
+  display: flex;
+  background-color: gray;
+  height: 80px;
+  color: white;
+  font-weight: bold;
+  margin: auto 0;
+  justify-content: flex-end;
+  align-items: center;
+  padding: 10px;
+  font-family: 'digital';
+  font-size: 50px;
```

Comment:
this font makes it like a real calculator display...i love it 

![Screen Shot 2020-04-01 at 4 45 30 PM](https://user-images.githubusercontent.com/40712459/78157299-b6f8bf80-7437-11ea-95fd-76008c2a9ccd.png)